packages/plugins/flow/flow/tests/__snapshots__/flow.spec.ts.snap

```diff
@@ -126,9 +126,9 @@ export type MyType = {|
 export type MyTypeFooArgs = {|
```

Comment:
I'm not 100% sure that we should modify `Args`. I think only `variables` should be effected. 

Let's say you have an argument defined for field, if you define it as `[String]`, your Args will always be `string[]`. But, on the level of consumption, where you use variables, you should be able to use `string | string[]`. 

What do you think?

Reply:
I actually don't know if GraphQL support coercion on `Args`
we should try :)

Reply:
Let's just apply it for `Variables` for now. I think `Args` will always be the way you define it.